common/lib/capa/capa/customrender.py

```diff
@@ -167,7 +167,9 @@ def __init__(self, system, xml):
         self.system = system
         # Get any text content found inside this tag prior to the first child tag. It may be a string or None type.
         initial_text = xml.text if xml.text else ''
-        self.inner_html = initial_text + ''.join(etree.tostring(element) for element in xml)
+        self.inner_html = initial_text.decode('utf-8') + ''.join(
+            etree.tostring(element).decode('utf-8') for element in xml
```

Comment:
I think `self.inner_html = initial_text + u''.join(etree.tostring(element, encoding='unicode') for element in xml)` might be a little cleaner ([lxml.etree API docs](https://lxml.de/tutorial.html#serialisation)).  It looks like the `text` and `tail` attributes are Unicode under Python 3, so I don't think they need to be decoded.

Thanks for tracking this down!